ros2bag/ros2bag/verb/play.py

```diff
@@ -116,6 +119,7 @@ def main(self, *, args):  # noqa: D102
         play_options.loop = args.loop
         play_options.topic_remapping_options = topic_remapping
         play_options.clock_publish_frequency = args.clock
+        play_options.delay=args.delay
```

Comment:
linter is failing on this line - it will help you to run the tests locally before putting up for review!

Reply:
Thaks for your review and advice. I modified it and checked that `test_flake8.py` paased. https://github.com/ros2/rosbag2/pull/789/commits/1c0cbffc9815858a106474661008c53df884312b